src/shared/components/Header/Navigation/styles.ts

```diff
@@ -55,11 +83,27 @@ export const ToggleMenu = styled.button`
     width: 100%;
     height: 2px;
     background-color: ${Colors.BLACK};
+    transition: all 0.25s ease-out;
   }
   &::after {
-    top: 6px;
+    ${({ isOpenMenu }: StyleProps): string =>
```

Comment:
May be you could try 
```
${p => `bottom: ${p.isOpenMenu ? 
```
```
${({ isOpenMenu }: StyleProps): string =>
`bottom: ${isOpenMenu ? 
```

Reply:
I have a mistake
<img width="973" alt="Screenshot 2020-04-22 at 12 04 01" src="https://user-images.githubusercontent.com/27691774/79962918-5cc8a880-8491-11ea-898a-8526d6dece97.png">

Reply:
This should probably help you https://stackoverflow.com/questions/57852188/missing-return-type-on-function-error-in-styled-component-interpolation-functi

Reply:
Done!